Sources/LiCore/Networking/LiRestClient.swift

```diff
@@ -21,7 +21,15 @@ typealias RefreshCompletion = (_ succeeded: Bool, _ error: Error?) -> Void
 
 class LiRestClient {
     static let sharedInstance = LiRestClient()
-    internal let sessionManager = SessionManager()
+    internal let sessionManager: SessionManager = {
+        let configuration = URLSessionConfiguration.default
+        var defaultHeaders = SessionManager.defaultHTTPHeaders
+        let webView = UIWebView(frame: .zero)
+        let secretAgent = webView.stringByEvaluatingJavaScript(from: "navigator.userAgent")
+        defaultHeaders["User-Agent"] = secretAgent
+        configuration.httpAdditionalHeaders = defaultHeaders
+        return SessionManager(configuration: configuration)
+    }()
```

Comment:
Seems like duplicate code.
I'm sure there's a way you could remove the redundancy.
Maybe have a custom class that does the initialization.